src/cli/cli_coap.cpp

```diff
@@ -83,109 +72,74 @@ void Coap::PrintPayload(otMessage *aMessage)
         }
     }
 
-    sServer->OutputFormat("\r\n");
-}
-
-void Coap::ConvertToLower(char *aString)
-{
-    uint8_t i = 0;
-
-    while (aString[i])
-    {
-        aString[i] = (char)tolower(aString[i]);
-        i++;
-    }
+    mServer->OutputFormat("\r\n");
 }
 
-void Coap::OutputBytes(const uint8_t *aBytes, uint8_t aLength)
+void Coap::OutputBytes(const uint8_t *aBytes, uint8_t aLength) const
 {
     for (int i = 0; i < aLength; i++)
     {
-        sServer->OutputFormat("%02x ", aBytes[i]);
+        mServer->OutputFormat("%02x ", aBytes[i]);
```

Comment:
Remove the trailing space in `%02x ` to keep consistent with `Interpreter::OutputBytes`, or just reuse it?